apps/livreGUI/Controller.cpp

```diff
@@ -21,238 +21,102 @@
  */
```

Comment:
So, with the below implementation different classes can not register different functions for the same event to the same subscriber. If this is the wanted behaviour I am not against that, but otherwise if there is another way to do that (same event can be registered with different functions to the same subscriber ) we should do that way I believe. Stefan told about subscribers can be added to other subscribers.